mlflow/protos/service.proto

```diff
@@ -246,6 +246,25 @@ service MlflowService {
     };
   }
 
+  // Set a tag on an experiment. Tags are run metadata that can be updated on an experiment.
```

Comment:
Let's change the second sentence to read `Experiment tags are metadata that can be updated.`.